### ISSUE TITLE
 Fail to scan the games

### ISSUE THREAD

**Issue by Newaser** [Ecks1337/RyuSAK#61]
My **Configuration** path is `D:\Program Files\Ryujinx\portable`, but it fails to scan any game.
It says"[Download Ryujinx](https://ryujinx.org/download) and launch any game first. All your games should show up here automatically after that. Please note, that only games launched in Ryujinx will be shown here. Ryujinx is not behind a paywall. If you are using a portable installation, use the "Configuration" button on the top left."

**Comment by yogithesymbian** [comment]
bro v1.6.2 arm64.dmg is damaged ... i alr 2x download its same problem . on release

**Comment by yogithesymbian** [comment]
> bro v1.6.2 arm64.dmg is damaged ... i alr 2x download its same problem . on release

solved by `xattr -cr /Applications/RyuSAK.app`

**Comment by 2O48** [comment]
Yes it can't scan any game though i see my games folder in the directory

**Comment by b4bass** [comment]
I cant scan and I cant see any game

**Comment by zixiaomiao** [comment]
me too ,I cant scan and I cant see any game

**Comment by eomantovani** [comment]
Did you guys find a solution for this?

**Comment by ShreyasPurohitcode** [comment]
I can't find my games either

**Comment by eomantovani** [comment]
Hi Guys, I found a way to resolve this issue.  Here are the steps: 

Step 1: On Ryujinx, go to File -> Open Ryujinx folder.  Take Note of that folder. Mine, since it is Mac, it is inside Library->Application Support-> Ryujinx. This folder should contain the file config.json. 

Step 2: Go to Ryusak , select "add additional configuration" and enter the same folder as step 1. 

Done!
